{{Introduction to Python Basics: Variables and Data types}}/.md

```diff
@@ -0,0 +1,180 @@
+Introduction to Python basics: Variables and Data Types
+Written by: Gisselle Ibarra
+
+
+Python is one of the most user friendly coding languages to learn as a beginner. Not only is it used in industry but also fun projects such as creating video games or even your own website. To create something more complicated like mentioned above we need to first learn the basics. That is why today we will be going over two basic Python concepts: Variables and Data Types. This blog will give basic understanding of these concepts. Throughout your reading I encourage you to try out all of the examples yourself and engage with the material if possible! In order to engage with the material we will need to use Python. Below is a link where you can download Python. If you are not able to do so it is okay! I will provide images and graphics. 
```

Comment:
If you want people to follow along, consider making this into a Jupyter Notebook or similar online executable code editor, or linking to one.